src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.Windows.cs

```diff
@@ -24,9 +24,11 @@ public static string OSDescription
             {
                 if (null == s_osDescription)
                 {
-                    // Temporarily commenting out code to prevent build break on mirror with TFS.
-                    // Will be changed from TFS mirror.
-                    s_osDescription = "Windows";/*Interop.NtDll.RtlGetVersion();*/
+#if !PLATFORM_NETCORE50
```

Comment:
nit: Any reason you did it this way (#if !<value>) instead of having the positive case first?